webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
+             double                        consentTotalRoundTripTime;
+             unsigned long long            consentRequestsReceived;
+             unsigned long long            consentRequestsSent;
+             unsigned long long            consentResponsesReceived;
```

Comment:
consentRequestsReceived/RequestsSent, etc. represent counts after initial consent is given.  Looking at this, it might make more sense to hvae consentRoundTripTime instead of consentTotalRoundTripTime.